examples/vpc-with-multiple-az/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "aws_region" {
+  description = "AWS region to deploy resources into"
+  type        = string
+  default     = "eu-central-1"
```

Comment:
Let's remove the defaults for the region and azs, @nikolaykolev complained about it, and he's right. We'll need to add instructions for the users to create `terraform.tfvars` file and set them there